code/modules/mob/new_player/new_player.dm

```diff
@@ -100,28 +84,32 @@
 				if(player.ready)totalPlayersReady++
 
 /mob/new_player/Topic(href, href_list) // This is a full override; does not call parent.
-	if(usr != src)
-		return TOPIC_NOACTION
-	if(!client)
+	if(usr != src || !client)
 		return TOPIC_NOACTION
 
-	if(href_list["show_preferences"])
+	if(href_list["lobby_changelog"])
+		client.changes()
+		return
+
+	if(href_list["lobby_setup"])
 		client.prefs.open_setup_window(src)
 		return 1
 
-	if(href_list["ready"])
-		if(GAME_STATE <= RUNLEVEL_LOBBY) // Make sure we don't ready up after the round has started
-			ready = text2num(href_list["ready"])
-		else
-			ready = 0
+	if(href_list["lobby_ready"])
+		if(ready && SSticker.pregame_timeleft <= 5 SECONDS)
+			to_chat(src, SPAN_WARNING("Locked! The round is about to start."))
+			return
```

Comment:
Why are we doing this? It's going to screw people over for no reason.

Reply:
Removed.